### DISCUSSION TITLE
when is 3.4.1 release?

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-workflows#9716] [Q&A] [answered]
I see a few regressions in 3.4.0 have been merged, most importantly the archived workflows ui fix

**Comment by fhochleitner** [comment]
@tooptoop4 it's already released. if you deployed via helm it's in chart version 0.20.1

**Comment by agilgur5** [comment, marked as answer]
It was officially released on Oct 1st, 2022: https://github.com/argoproj/argo-workflows/releases/tag/v3.4.1.
